salt/tests/unit/modules/test_cordon.py

```diff
@@ -0,0 +1,125 @@
+from parameterized import parameterized
+
+import salt
+from salt.exceptions import CommandExecutionError
+
+from salttesting.mixins import LoaderModuleMockMixin
+from salttesting.unit import TestCase
+from salttesting.mock import MagicMock, patch
+
+import metalk8s_cordon
+
+
+class CordonTestCase(TestCase, LoaderModuleMockMixin):
+    """
+    TestCase for `cordon` module
+    """
+    loader_module = metalk8s_cordon
+
+    def node_object(self, unschedulable=None, with_taints=False):
+        node_obj_dict = {
+            'api_version': 'v1',
+            'kind': 'Node',
+            'metadata': {
+                'name': 'bootstrap'
+            },
+            'spec': {
+                'taints': [{
+                    'effect': 'NoSchedule',
+                    'key': 'node-role.kubernetes.io/bootstrap'
+                }],
+                'unschedulable': unschedulable
+            }
+        }
+        if with_taints:
+            node_obj_dict['spec']['taints'].append({
+                'effect':
+                'NoSchedule',
+                'key':
+                'node.kubernetes.io/unschedulable'
```

Comment:
Formatting is wrong here, it's a dict

```suggestion
                'effect': 'NoSchedule',
                'key': 'node.kubernetes.io/unschedulable'
```